packages/aws-lambda/src/wrapper.js

```diff
@@ -432,9 +433,9 @@ function postHandler(entrySpan, error, result, callback) {
   const spans = spanBuffer.getAndResetSpans();
 
   // We want that all upcoming spans are send immediately to the BE.
-  // Span collection happens all the time and they are buffered via spanBuffer.
-  // The spanBuffer will only send out the spans early if the payload becomes too big.
-  // When the Lambda handler finishes we trigger `sendBundle`.
+  // Span collection happens all the time, but for AWS Lambda sending spans early via spanBuffer
```

Comment:
Qs: For AWS Lambda we are not using spanBuffer?

Reply:
Exactly. The comment describes why.

Reply:
Great! I'm glad you found this and updated the comments ❤️